jaeger-thrift/build.gradle

```diff
@@ -42,8 +42,11 @@ checkstyleTest.enabled = false
 
 shadowJar {
     baseName = 'jaeger-thrift'
-    relocate 'org.apache.thrift', 'org.shadow.apache.thrift92'
-    relocate 'okhttp', 'jaeger.okhttp'
+    relocate 'com.google.gson'  , 'jaeger.com.google.gson'
+    relocate 'com.twitter'      , 'jaeger.com.twitter'
+    relocate 'okhttp'           , 'jaeger.okhttp'
+    relocate 'okio'             , 'jaeger.okio'
+    relocate 'org.apache'       , 'jaeger.org.apache'
     classifier 'shadow'
```

Comment:
I thought the idea was to make it the main artifact, not have a classifier?

Reply:
Good point, I think that's indeed what we discussed last Friday.

Reply:
Created #460 and #461